services/app-api/forms/mcpar.json

```diff
@@ -224,20 +224,20 @@
           }
         },
         {
-          "name": "Add In Lieu of Services",
-          "path": "/mcpar/program-information/add-in-lieu-of-services",
+          "name": "Add In Lieu of Services and Settings",
+          "path": "/mcpar/program-information/add-in-lieu-of-services-and-settings",
           "pageType": "standard",
           "verbiage": {
             "intro": {
               "section": "Section A: Program Information",
-              "subsection": "In Lieu of Services and Settings Offered",
+              "subsection": "Add In Lieu of Services and Settings Offered",
               "info": [
                 {
                   "type": "p",
                   "children": [
                     {
                       "type": "html",
-                      "content": "If In Lieu of Services and Settings (ILOS) are available in this managed care program, enter the name of each ILOS offered, as approved in the managed care plan contract(s). Do not include short-term IMD stays. <b>You are required to complete this section if ILOS are available.</b> Do not include short-term IMD stays. Access "
+                      "content": "If In Lieu of Services and Settings (ILOS) are available in this managed care program, Enter the name of each ILOS, as approved in the managed care plan contract(s). <b>Complete this section if ILOS are available in your program.</b> Access "
```

Comment:
I think we're trying to move away from using `html` in favor of `text`? At least that's what I remember Jess commenting on a previous PR of mine in MFP.

Reply:
@jessabean can you confirm? 👀

Reply:
I think the markup here is right...the parent on line 236 is `p` and this is necessary to combine the regular text content with the `externalLink.` 

My teeny tiny nit for this section is that I would change the `<b>` to `<strong>` in this block of text 😁